pipelines/shopgate.user.loginUser.v1.json

```diff
@@ -14,12 +14,22 @@
     "steps": [
       {
         "type": "extension",
-        "id": "@shopgate/user",
```

Comment:
I can't comment on a proper line as it is too far from changes. The tracking can't be part of this pipeline. It needs to be supported in another story. There is a comment about it in related [ticket](https://bit.ly/2IxshMF).

Reply:
Removed.

Reply:
ok :-)